server/odc-service/src/main/java/com/oceanbase/odc/service/resultset/ResultSetExportTask.java

```diff
@@ -220,7 +220,7 @@ private void setColumnConfig(DataTransferConfig config, ResultSetExportTaskParam
         List<MaskingAlgorithm> algorithms = parameter.getRowDataMaskingAlgorithms();
         Map<String, Map<String, List<OrdinalColumn>>> catalog2TableColumns = new HashMap<>();
         try {
-            SyncJdbcExecutor syncJdbcExecutor = session.getSyncJdbcExecutor(ConnectionSessionConstants.BACKEND_DS_KEY);
+            SyncJdbcExecutor syncJdbcExecutor = session.getSyncJdbcExecutor(ConnectionSessionConstants.CONSOLE_DS_KEY);
```

Comment:
this datasource may be occupied~

Reply:
This session was constructed by result export task, and there is no reuse. So there will be no conflict.